### DISCUSSION TITLE
Phone recommendations for running xDrip+

### DISCUSSION THREAD

**Discussion by DapperDom** [NightscoutFoundation/xDrip#2625] [Q&A]
Hi all,

Wondering if I can lean on some of the collective experience here?!

I'm a fairly newly minted type 1, have now -after a couple of months of tweaking - got xDrip+ running pretty nicely on my current phone (a OnePlus 7 Pro).

However, it's about time to replace it (screen pretty cracked up too!), and have the opportunity to get a device through work, so trying to find out what the best phone to get would be - this is really kind of it's primary purpose now, so a big factor in choosing a new handset.

Given that I've got my current phone working quite well, I'm considering a OnePlus 10 Pro, but keen to hear if anyone has any advice to share before taking the plunge...

Any good or bad experiences with that specific device?  Any brands particularly good or bad in general?  (Looks like Samsung is one to avoid?).  Any particular model anyone would advise as being especially well suited to xDrip+?

I'm using the Libre 2 sensors over BT, if that makes any difference...

Any and all info and input very gratefully welcomed!

And also, a HUGE thanks to everyone who's contributed to making this software, game changing is an overused phrase but this really is totally game changing.  Really awesome, thank you all so much...

Cheers,
Dom

**Comment by Navid200** [comment]
I have heard about issues with Xiaomi.

**Reply by DapperDom** [reply]
Thanks Navid...  I actually had a Xaomi years ago, and that kind of fits with my experience - the hardware was pretty good, but the software had lots of "gaps" in terms of dealing properly with stuff like security and connectivity.

Not really designed for the UK market perhaps?

I think I'm going to go for the latest version of what I've got (so a OnePlus 10 Pro), as (a) the current setup works well enough and so hopefully that'll still be the case on a newer device, and (b) from searching around I see quite a few people happy with other OnePlus devices (no direct info on that particular model I can find, though).

I'll report back!
